scripts/package-next

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -eux
 
+npm install -g npm@11
```

Comment:
I think we do not need this. The `prepare` job should set npm for the CI and we should enforce using npm `>= v11.5.1` in the `package.json` using the `engine` field. Field we should probably remove when we publish the library. I propose that we deal with this in a separate PR.

Reply:
Ah I see. `next` should probably also uses `prepare`. Ok then let's add the line here and clean-up the action afterwards.